app/views/notifications_mailer/project_invite.html.erb

```diff
@@ -1,7 +1,18 @@
-<h1>Project invite</h1>
+<%
```

Comment:
This feels like it might be better in the mailer object to limit data/logic in the view.

Reply:
Yeah, I was thinking about this. I ended up putting it here because it felt very view-specific, but if we're going to reuse this text somewhere else then I agree it should be defined centrally (maybe on the membership model?).

Reply:
I'd probably stick it in [NotificationsMailer#project_invite](https://github.com/TechforgoodCAST/fusebox-insights/blob/dev/app/mailers/notifications_mailer.rb#L4-L7), but where it is also works.